Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -251,10 +251,14 @@ class SettingsViewController: TableViewController {
             )
         }
         
-        general.rows.append(
-            .boolRow(title: Strings.enablePullToRefresh,
-                     option: Preferences.General.enablePullToRefresh,
-                     image: #imageLiteral(resourceName: "settings-pull-to-refresh").template)
+        general.rows.append(contentsOf: [
+                .boolRow(title: Strings.enablePullToRefresh,
+                         option: Preferences.General.enablePullToRefresh,
+                         image: #imageLiteral(resourceName: "settings-pull-to-refresh").template),
+                .boolRow(title: Strings.mediaAutoPlays,
```

Comment:
I'm not 100% sure, but I think we need to add:
`Preferences.General.mediaAutoPlays.observe(from: self)` to `fileprivate func didInit()`>

Reply:
Seems to work fine without it, UI updates correctly as well as the preference value